### PR TITLE
BAVet145

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: VaticanBAV/et/BAVet120.xml [hunk 1]
```diff
@@ -107,7 +107,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <locus target="#42v"></locus>
 <desc type="Subscription"></desc>
 <q xml:lang="gez">ዝ፡ መጽሐፍ፡ ዘ<placeName ref="LOC1713Bagemd">በጌምድር፡</placeName>
- <placeName ref="LOC3056Farta">ፋርጣ፡</placeName> ዓዲኮ፡ <persName role="owner" ref="PRS12979GabrielFA">ገብርኤል</persName>፡
+    <placeName ref="LOC3056Farta">ፋርጣ፡</placeName> ዓዲኮ፡ <placeName ref="INS1013FAG">ገብርኤል</placeName>፡
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet120.xml:110]: I would only markup `<placeName ref="INS1013FAG">ፋርጣ፡ ዓዲኮ፡ ገብርኤል</placeName>`, the information on warada and qabale is for the settlement section of the INS1013FAG record

File: VaticanBAV/et/BAVet145.xml [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+                        </handDesc>
+                      
+                      <additions>
+                        <list>
+                          <item xml:id="a1">
+                            <locus target="#74v"/>
+                            <desc type="OwnershipNote">Ownership note stating the name of the donor, <persName ref="PRS12978GabraSelassie">Gabra Śǝllāse</persName>, and of 
+                            members of his family.</desc>
+                            <q xml:lang="gez">ዝመጽሐፍ፡ <placeName ref="LOC1713Bagemd"><sic>ዘለጌምድር፡</sic></placeName> <placeName ref="LOC3056Farta">ፋርጣ፡</placeName> ዓዲኮ፡ 
+                              <placeName ref="INS1013FAG">ገብር<lb/>ኤል፡</placeName> ዘወሀቦ፡
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet145.xml:131]: as above

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

File: VaticanBAV/et/BAVet145.xml [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+                          </handNote>
+                        </handDesc>
+                      
+                      <additions>
+                        <list>
+                          <item xml:id="a1">
+                            <locus target="#74v"/>
+                            <desc type="OwnershipNote">Ownership note stating the name of the donor, <persName ref="PRS12978GabraSelassie">Gabra Śǝllāse</persName>, and of 
+                            members of his family.</desc>
+                            <q xml:lang="gez">ዝመጽሐፍ፡ <placeName ref="LOC1713Bagemd"><sic>ዘለጌምድር፡</sic></placeName>
```
Comment (DenisNosnitsin1970) [inline VaticanBAV/et/BAVet145.xml:130]: this is definitely በ.

File: VaticanBAV/et/BAVet145.xml [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+                              ባለ፡ ዝንቱ፡ መ<lb/>ካን፡ ዓዲኮ፡ 
+                              ወስመ፡ እኁሁ፡ <persName ref="PRS12980WaldaGiyrgis">ወልደ፡ ጊዮርስ፡</persName> 
+                              ወ<lb/>እሙ፡ <persName ref="PRS13011WalattaGabriel">ወለተ፡ ገብርኤል፡</persName>
+                              ከመ፡ ይኩኖሞ፡ ተዝ<lb/>ካረ፡ እስከ፡ ለዓለመ፡ ዓለም።</q>
+                          </item>
+                          
+                          <item xml:id="a2">
+                            <locus target="#5r"/>
+                            <desc type="Unclear">According to the catalogue, this note alludes to the <ref target="#a1">ownership note</ref>.</desc>
+                            <q xml:lang="am">፯ተኛው፡ ጥራዝ፡ <supplied reason="omitted">የ</supplied>ሚፈጽም፡ ደዳቤውን፡ ፈልጎ፡</q>
```
Comment (DenisNosnitsin1970) [inline VaticanBAV/et/BAVet145.xml:144]: please add ብ after ደ

File: VaticanBAV/et/BAVet145.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply VaticanBAV/et/BAVet145.xml:130]: The sense of the note is escaping me :)

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**